README.md

```diff
@@ -1094,8 +1093,8 @@ before_install:
   - wget https://github.com/aquasecurity/trivy/releases/download/v${VERSION}/trivy_${VERSION}_Linux-64bit.tar.gz
   - tar zxvf trivy_${VERSION}_Linux-64bit.tar.gz
 script:
-  - ./trivy --exit-code 0 --severity HIGH --quiet --auto-refresh trivy-ci-test:${COMMIT}
-  - ./trivy --exit-code 1 --severity CRITICAL --quiet --auto-refresh trivy-ci-test:${COMMIT}
+  - ./trivy --exit-code 0 --severity HIGH -no-progress --auto-refresh trivy-ci-test:${COMMIT}
```

Comment:
```suggestion
  - ./trivy --exit-code 0 --severity HIGH --no-progress --auto-refresh trivy-ci-test:${COMMIT}
```
I know that `-no-progress` works as well.

Reply:
Sorry, I misspelled it.